### PR TITLE
feat: sync select with design system

### PR DESCRIPTION
**Pull request by jose-costa-frontify**
CU-8695bwrp1

### CODE REVIEW

File: packages/components/src/components/Select/Combobox.tsx [hunk 1]
```diff
@@ -88,15 +94,26 @@ export const SelectCombobox = (
...
 
+    const localStatus = useMemo(() => {
+        if (status === 'error' || valueInvalid) {
+            return 'error';
+        }
+        if (status === 'success') {
+            return 'success';
+        }
+        return 'neutral';
+    }, [status, valueInvalid]);
```
Comment (SamuelAlev) [inline packages/components/src/components/Select/Combobox.tsx:111]: Why do you need this?

File: packages/components/src/components/Select/Combobox.tsx [hunk 2]
```diff
@@ -88,15 +94,26 @@ export const SelectCombobox = (
...
+
     return (
         <RadixPopover.Root open={isOpen}>
             <RadixPopover.Anchor asChild>
-                <div ref={forwardedRef} className={styles.root} data-error={valueInvalid}>
+                <div ref={forwardedRef} className={styles.root} data-status={localStatus}>
```
Comment (SamuelAlev) [inline packages/components/src/components/Select/Combobox.tsx:116]:
```suggestion
                <div ref={forwardedRef} className={styles.root} data-status={valueInvalid || status}>
```

File: packages/components/src/components/Select/Combobox.tsx [hunk 3]
```diff
@@ -135,17 +152,33 @@ export const SelectCombobox = (
...
+                            <IconCaretDown size={16} className={styles.caret} />
+                        </button>
+                        {status === 'success' ? (
+                            <IconCheckMark
+                                size={16}
+                                className={styles.iconSuccess}
+                                data-test-id={`${dataTestId}-success-icon`}
+                            />
+                        ) : null}
+                        {status === 'error' ? (
```
Comment (SamuelAlev) [inline packages/components/src/components/Select/Combobox.tsx:174]:
```suggestion
                        {valueInvalid || status === 'error' ? (
```

File: packages/components/src/components/Select/Combobox.tsx (hunk 2 above)
Comment (jose-costa-frontify) [reply packages/components/src/components/Select/Combobox.tsx:116]: this does seem simpler come to think of it 😅

File: packages/components/src/components/Select/Combobox.tsx (hunk 3 above)
Comment (jose-costa-frontify) [reply packages/components/src/components/Select/Combobox.tsx:174]: right!

File: .changeset/sharp-fireants-know.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue-components": patch
+---
+
+feat: update styles
```
[the remark this answers is not in the document]
Comment (jose-costa-frontify) [reply .changeset/sharp-fireants-know.md:5]: yep, oversight

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**